tests/python/unittest/test_tir_transform_compact_buffer_region.py

```diff
@@ -505,6 +505,74 @@ def compacted_opaque_access_annotated_func(a: T.handle) -> None:
                 T.store(C.data, i, T.load("float32", B.data, i))
 
 
+@T.prim_func
+def sparse_read_cache(
+    A_data: T.Buffer[(819,), "float32"],
+    B: T.Buffer[(128,), "float32"],
+    A_indptr: T.Buffer[(129,), "int32"],
+    A_indices: T.Buffer[(819,), "int32"],
+) -> None:
+    for i in T.serial(128):
+        with T.block("rowsum_outer"):
+            T.reads(
+                A_indptr[i],
+                A_indptr[i + 1],
```

Comment:
IIRC, we don't allow annotating one buffer twice in `read` and `write` regions. Here it's better to be `A_indptr[i: i + 1]`

Reply:
Ohhh thanks for the reminder! Will fix it soon.

Reply:
@Hzfengsy So there is no way we can do `T.reads(A_indptr[i], A_indptr[i + 2])`?

Reply:
I suppose the only way is `T.reads(A_indptr[i : i+2])` - writing read-regions like this is safe. But I have no idea how to represent `T.writes(data[i], data[i + 2])` well... Asserting an overlarge write-region is not equally safe.

Reply:
Okay that make sense.